README.md

```diff
@@ -38,7 +40,32 @@ Widget build(BuildContext context) {
 }
 ```
 
-## Configuration
+#### Mutation
+
+```dart
+VoltMutation<String> useDeletePhotoMutation() {
```

Comment:
I think the example for this should be the simplest version of using this meaning that the useMutation is used inline in the build function